### PR TITLE
Add documentation for ToAnotherOrDefault

### PR DESCRIPTION
**Pull request by Kurnakova**

### CODE REVIEW

**Review by kurnakovv — CHANGES_REQUESTED** [review]

**Review by kurnakovv — CHANGES_REQUESTED** [review]

File: DocsReadMe.md [hunk 1]
```diff
@@ -130,6 +130,59 @@ bool isConverted = invalidSecondValue.TryToAnother(out AnotherEnum anotherEnum);
 // [isConverted, anotherEnum] - [false, First]
 ```
 
+# ToAnotherOrDefault
+
+**Description**
+
+Convert input enum to another enum or default value if not possible.
```
Comment (kurnakovv) [inline DocsReadMe.md:137]: Hmm, lets add more details
input enum -> `enumValue`
another enum -> `TAnotherEnum`
default value -> `defaultValue`

File: DocsReadMe.md [hunk 2]
```diff
@@ -130,6 +130,59 @@ bool isConverted = invalidSecondValue.TryToAnother(out AnotherEnum anotherEnum);
...
+public enum AnotherEnum { First, Second, Third }
+
+public enum InputEnum { First, Second, Third }
+public enum InvalidEnum { FirstInvalidValue, SecondInvalidValue }
+
+
+// Input enums
+InputEnum inputEnumFirst = InputEnum.First;
+InvalidEnum invalidSecondValue = InvalidEnum.SecondInvalidValue;
+AnotherEnum defaultValue = AnotherEnum.Third;
```
Comment (kurnakovv) [inline DocsReadMe.md:169]: rename `defaultValue` -> `customDefaultValue` and for tests too

**Review by kurnakovv — CHANGES_REQUESTED** [review]

File: DocsReadMe.md (hunk 1 above)
Comment (Kurnakova) [reply DocsReadMe.md:137]: Then it would be different from other descriptions in DocsReadMe file. Should I change them too?

Comment (kurnakovv) [reply DocsReadMe.md]: Oh well, you're right... 
Sorry, let's revert this changes and will follow single style

**Review by kurnakovv — APPROVED** [review]
@Kurnakova LGTM, thank you very much :)

**Pull request merged**